python/tk_desktop/desktop_window.py

```diff
@@ -504,9 +503,11 @@ def show_update_project_config(self):
         self.update_project_config_widget.show()
         self.project_overlay.hide()
 
+    def _get_current_user(self):
+        return login.get_current_user(sgtk.platform.current_engine().tank)
```

Comment:
`get_current_user()` tries to match the operating system user right? Should we add a comment here that this is going to be fixed soon? Or is it ok this way?